### ISSUE TITLE
Use splitter for UI output and console panels

### ISSUE THREAD

**Issue by johnpryan** [dart-lang/dart-pad#2789]
The console output can be hard to read since it's not resizable:

![Screenshot 2024-01-09 at 10 22 17 AM](https://github.com/dart-lang/dart-pad/assets/1145719/ca437e4a-6720-4d30-8b67-e39d32a3a55b)

**Comment by parlough** [comment]
Since we don't currently show the console if running a Flutter app, if that's what we want to keep doing, perhaps the console should replace the app view if there was a compilation failure?

**Comment by johnpryan** [comment]
I like the expandable console we currently have - this idea came from the results of a UX study so we might want to consider keeping this behavior.

**Comment by johnpryan** [comment]
Yeah that seems like we omitted that feature by mistake out of expediency. Flutter logs contain a lot of information but they should still be made available somehow IMO.
